utils/build-script-impl

```diff
@@ -984,17 +984,17 @@ fi
 if [[ ! "${SKIP_BUILD_LLBUILD}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" llbuild)
 fi
-if [[ ! "${SKIP_BUILD_SWIFTPM}" ]] ; then
-     PRODUCTS=("${PRODUCTS[@]}" swiftpm)
-fi
 if [[ ! "${SKIP_BUILD_LIBDISPATCH}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" libdispatch)
 fi
-# XCTest has a dependency on Foundation, so Foundation must be added to the
-# list of build products first.
+# SwiftPM and XCTest have a dependency on Foundation, so Foundation must be
+# added to the list of build products first.
 if [[ ! "${SKIP_BUILD_FOUNDATION}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" foundation)
 fi
+if [[ ! "${SKIP_BUILD_SWIFTPM}" ]] ; then
+     PRODUCTS=("${PRODUCTS[@]}" swiftpm)
+fi
```

Comment:
Let's go ahead and build SwiftPM after XCTest here, since it also uses XCTest even if it doesn't need it for the initial build currently.